cmd/minishift/cmd/start.go

```diff
@@ -414,6 +416,16 @@ func validateOpenshiftVersion() {
 func setSubscriptionManagerParameters() {
 	cluster.RegistrationParameters.Username = viper.GetString(startFlags.Username.Name)
 	cluster.RegistrationParameters.Password = viper.GetString(startFlags.Password.Name)
+	cluster.RegistrationParameters.GetUsernameInteractive = getUsernameInteractive
+	cluster.RegistrationParameters.GetPasswordInteractive = getPasswordInteractive
+}
+
+func getUsernameInteractive() string {
```

Comment:
Right, except this method should take a parameter as well, the message. This is what got us here, remember? We wanted to have more specific messages when asking for credentials. For example now we can say "Please provide your RHEL subscription username" (or whatever @LalatenduMohanty mentioned in an earlier comment).